### PR TITLE
add an optional aws-builds job on origin

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@stevekuznetsov please sanity check, my intent is that this should not block anyone right now.

**Comment by wking** [comment]
`optional: true` and `always_run: false` are enough guards for me ;).

/lgtm

File: ci-operator/jobs/openshift/origin/openshift-origin-master-presubmits.yaml [hunk 1]
```diff
@@ -123,6 +123,63 @@ presubmits:
...
+      volumes:
+      - name: cluster-profile
+        projected:
+          sources:
+          - secret:
+              name: cluster-secrets-aws
+      - configMap:
+          name: prow-job-cluster-launch-installer-e2e
+        name: job-definition
+    trigger: ((?m)^/test( all| e2e-aws-builds),?(\s+|$))
```
[the remark this answers is not in the document]
Comment (wking) [reply ci-operator/jobs/openshift/origin/openshift-origin-master-presubmits.yaml]: Oh well, looks like the generator won't let us do this.

/hold cancel

[the remark this answers is not in the document]
Comment (bparees) [reply ci-operator/jobs/openshift/origin/openshift-origin-master-presubmits.yaml]: generator won't let it be removed.  reverted back to including "All"

**Pull request merged**